.github/workflows/gh-container-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Update container image
+
+on:   
+  push:
+    branches:
+      - main
+
+env:
+  PROCESSOR_NAME: ghcr.io/${{ github.repository_owner }}/blockexplorer-processor:latest
+  API_SERVER_NAME: ghcr.io/${{ github.repository_owner }}/blockexplorer-api-server:latest
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build processor image
+        run: docker build . --target processor -t squid-processor --tag $PROCESSOR_NAME
+        working-directory: squid-blockexplorer
+
+      - name: Build API server image
+        run: docker build . --target query-node -t graphql-server --tag $API_SERVER_NAME
+        working-directory: squid-blockexplorer
+
+      - name: Log in to registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
minor nitpick: should we move this step before `docker build` steps? In the event there's some login issue, we can save time failing here instead of after building the images.